src/functions/template-tags/general.php

```diff
@@ -1355,6 +1355,13 @@ function tribe_events_get_the_excerpt( $post = null, $allowed_html = null ) {
 		 */
 		$allow_shortcode = apply_filters( 'tribe_events_excerpt_allow_shortcode', false );
 
+		/**
+		 * Prevent the removal of shortcode markup in the Excerpt
+		 *
+		 * @var bool
+		 */
+		$remove_shortcode = apply_filters( 'tribe_events_excerpt_prevent_shortcode_removal', false );
```

Comment:
It confuses me a little that the hook is named _*prevent* shortcode removal_ and defaults to false while down on line 1396 the conditional is inverted (ie, we *do* remove anything looking like a shortcode, if it is false).